src/signals/settings/users/containers/Detail/hooks/__tests__/useFetchUser.test.js

```diff
@@ -1,24 +1,28 @@
-import { renderHook } from '@testing-library/react-hooks';
+import { renderHook, act , cleanup} from '@testing-library/react-hooks';
 import userJSON from 'utils/__tests__/fixtures/user.json';
-import useFetchUser, { userEndpoint } from '../useFetchUser';
+import useFetchUser from '../useFetchUser';
 
 describe('signals/settings/users/containers/Detail/hooks/useFetchUser', () => {
-  beforeEach(() => {
+  afterEach(() => {
+    cleanup();
```

Comment:
die hoeft niet meer toch?

Reply:
This function comes from the `hooks` lib and not from the testing library